File: src/ProductSaleServices/EmailAPI/src/main/java/com/buyte/EmailAPI/EmailData/PurchaseData.java
package com.buyte.EmailAPI.EmailData;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class PurchaseData {
    private String productName;
    private String productCondition;
    private String productDescription;
    private String quantity;

    private String toEmail;
}

File: src/ProductSaleServices/ProductService.DAL/DTOs/PurchaseData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProductService.DAL.DTOs
{
    public class PurchaseData
    {
        public string ProductName { get; set; }
        public string ProductCondition { get; set; }
        public string ProductDescription { get; set; }
        public string Quantity { get; set; }
        public string ToEmail { get; set; }

    }
}
